### PR TITLE
fix: change period to comma

### PR DESCRIPTION
**Pull request by yokishava**
### Why
The character before omitempty in the RichTextSectionElement struct tag was period.

### What
I changed to comma.

### CODE REVIEW

File: block_rich_text.go [hunk 1]
```diff
@@ -294,7 +294,7 @@ func NewRichTextSectionLinkElement(url, text string, style *RichTextSectionTextS
 type RichTextSectionTeamElement struct {
 	Type   RichTextSectionElementType `json:"type"`
 	TeamID string                     `json:"team_id"`
-	Style  *RichTextSectionTextStyle  `json:"style.omitempty"`
+	Style  *RichTextSectionTextStyle  `json:"style,omitempty"`
```
Comment (kanata2) [inline block_rich_text.go:297]: Thanks 🙏

File: block_rich_text.go [hunk 2]
```diff
@@ -312,64 +312,76 @@ func NewRichTextSectionTeamElement(teamID string, style *RichTextSectionTextStyl
...
 	UsergroupID string                     `json:"usergroup_id"`
+	Style       *RichTextSectionTextStyle  `json:"style,omitempty"`
 }
 
 func (r RichTextSectionUserGroupElement) RichTextSectionElementType() RichTextSectionElementType {
 	return r.Type
 }
 
-func NewRichTextSectionUserGroupElement(usergroupID string) *RichTextSectionUserGroupElement {
+func NewRichTextSectionUserGroupElement(usergroupID string, style *RichTextSectionTextStyle) *RichTextSectionUserGroupElement {
```
Comment (kanata2) [inline block_rich_text.go]: This breaks backward-compatibility, so if you want to use this immediately, maybe divide this PR into another one.

Comment (kanata2) [reply block_rich_text.go]: (If there is nothing wrong with including next minor release, you don't divide.)

Comment (yokishava) [reply block_rich_text.go]: Thanks for the review.
You are right about breaking backward compatibility.  This PR should only be a comma modification of `omitempty`, and the style addition changes should be created as a new separate PR.

Comment (yokishava) [reply block_rich_text.go]: PR was changed to comma change only.
The title and description of the PR have also been changed.

**Pull request merged**